src/AaveTokenV3.sol

```diff
@@ -391,7 +391,28 @@ contract AaveTokenV3 is BaseAaveTokenV2, IGovernancePowerDelegationToken {
     override
   {
     address delegatee = _getDelegateeByType(delegator, _balances[delegator], delegationType);
-    require(delegatee == msg.sender, 'INVALID_CALLER');
+    require(delegatee == msg.sender, 'NO_DELEGATION');
     _delegateByType(delegator, delegator, delegationType);
   }
+
+  /// @inheritdoc IGovernancePowerDelegationToken
+  function renounceDelegator(address delegator) external override {
+    address votingDelegatee = _getDelegateeByType(
+      delegator,
+      _balances[delegator],
+      GovernancePowerType.VOTING
+    );
+    address propositionDelegatee = _getDelegateeByType(
+      delegator,
+      _balances[delegator],
+      GovernancePowerType.PROPOSITION
+    );
+    require(votingDelegatee == msg.sender || propositionDelegatee == msg.sender, 'NO_DELEGATION');
```

Comment:
@eboadom "or" or "and", interesting question

Reply:
AND of course. OR is a bug

Reply:
Why though?
I perceived this to be the lazy function a la `renounce Whatever`, why would i fail if i only had one delegation?
Seems like a waste of gas as it's clear that the user wanted to renounce.

Reply:
Ah, that is another way of seeing it yes. I think if documented, makes sense, don't see where it could be a problem to have OR behaviour

Reply:
Let's merge then